martus-client/source/org/martus/client/swingui/dialogs/PureFxScrollableTextDlg.java

```diff
@@ -90,10 +96,18 @@ public PureFxScrollableTextDlg(UiMainWindow owner, String titleTag, String okBut
 			}
 
 			TextArea details = createTextArea(TokenReplacement.replaceTokens(text, tokenReplacement));
+			details.setPrefHeight(DETAILS_PREF_HEIGHT);
 			mainBox.getChildren().add(details);
 
 			setResultConverter(dialogButton -> dialogButton == okButton);
 
+			if (bottomPanel != null)
```

Comment:
The if statement can be extracted out into a method, then a subclass class PureFxScrollableTextDlgWithBottomPanel can override the method and add the buttom panel.  Creators of PureFxScrollableTextDlg will also not pass null.